packages/ra-ui-materialui/src/button/BulkDeleteWithUndoButton.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ActionDelete from '@material-ui/icons/Delete';
-import colorManipulator, { makeStyles } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
```

Comment:
Seems like it's discouraged to do 3rd level imports, , check the link

    @material-ui/core/styles/colorManipulator
                   1     2           3
https://material-ui.com/guides/minimizing-bundle-size/

![LevelImports](https://user-images.githubusercontent.com/4523129/71999711-35cbcb00-3242-11ea-808b-0236694443bc.png)

Reply:
You are seen the first commit. In the last one, I change it.

I saw that problem, but unless you have a solution, I have to leave it like it was because you cannot import `colorManipulator` directly.

Reply:
Sorry!

Reply:
It's ok.